README.md

```diff
@@ -772,6 +772,26 @@ so the examples can be tested automatically by MDX.
 
 ## Filesystem Access
 
+<!--
+Cleanup previous runs due to [dune runtest --watch] not doing it
+```ocaml
+Eio_main.run @@ fun env ->
+let rec rm_r path =
```

Comment:
Would probably make sense to add this to `Eio.Path` (Python calls it [rmtree](https://docs.python.org/3/library/shutil.html#shutil.rmtree)).